x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -21,6 +21,8 @@ export const KUBERNETES_RUN_INSTRUCTIONS =
 export const STANDALONE_RUN_INSTRUCTIONS_LINUXMAC = 'sudo ./elastic-agent install';
 export const STANDALONE_RUN_INSTRUCTIONS_WINDOWS = '.\\elastic-agent.exe install';
 
+export const FLEET_ELASTIC_AGENT_DETAILS_DASHBOARD_ID =
+  'elastic_agent-f47f18cc-9c7d-4278-b2ea-a6dee816d395';
```

Comment:
is it safe to hardcode the dashboard id, can it ever change?

Reply:
Yes unfortunately we do not have a better way to get a specific dashboard.

Reply:
we link to dashboards from the data streams page, maybe something similar can be done here?